lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -0,0 +1,60 @@
+require 'govuk_navigation_helpers'
+
+module GovukPublishingComponents
+  module Presenters
+    # @private
+    class ContextualNavigation
+      attr_reader :content_item, :request_path
+
+      # @param content_item A content item hash with strings as keys
+      # @param request_path `request.path`
+      def initialize(content_item, request_path)
+        @content_item = content_item
+        @request_path = request_path
+      end
+
+      def taxonomy_sidebar
+        nav_helper.taxonomy_sidebar
+      end
+
+      def taxon_breadcrumbs
+        nav_helper.taxon_breadcrumbs
+      end
+
+      def breadcrumbs
+        if content_item["schema_name"] == "specialist_document"
```

Comment:
I know this is ported functionality but it would be great to get this out of here soon especially as the advanced search finder will have custom breadcrumb styling.

Reply:
👍